source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,135 +4,382 @@ Sort Results
+``Bson`` interface and specify sort criteria as arguments. For information
+on the ``Bson`` interface, see our 
+:doc:`guide on BSON </fundamentals/data-formats/document-data-format-bson/>`.   
+
+You can use the ``sort()`` method of a ``FindIterable`` instance as follows:
 
 .. code-block:: java
 
-    collection.insertMany(Arrays.asList(
-        new Document().append("_id", 1)
-            .append("title", "The Brothers Karamazov").append("length", 824)
-            .append("author", "Dostoyevsky"),
-        new Document().append("_id", 2)
-            .append("title", "Les Misérables").append("length", 1462).append("author", "Hugo"),
-        new Document().append("_id", 3)
-            .append("title", "Atlas Shrugged").append("length", 1088).append("author", "Rand"),
-        new Document().append("_id", 4)
-            .append("title", "Infinite Jest").append("length", 1104).append("author", "Wallace"),
-        new Document().append("_id", 5)
-            .append("title", "Cryptonomicon").append("length", 918).append("author", "Stephenson"),
-        new Document().append("_id", 6)
-            .append("title", "A Dance with Dragons").append("length", 1104)
-            .append("author", "Martin")
-    ));
-
-Call the ``sort()`` function and specify the order with ``ascending()`` to ensure that the
-operation returns books in alphabetical order by title:
+   import static com.mongodb.client.model.Sorts.ascending;
 
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("_id"));
+ 
+You can use the ``Aggregates.sort()`` method as follows:
+   
 .. code-block:: java
-  :emphasize-lines: 5
 
-    import static com.mongodb.client.model.Sorts.ascending;
+   import com.mongodb.client.model.Aggregates;
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.aggregate(Arrays.asList(Aggregates.sort(ascending("_id"))));
+
+The code snippets above both sort the documents in the
+:ref:`sort_example collection <sorts-crud-sort-example>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   ...
+
+In the above code snippets, we specify our sort criteria using the ``Sorts``
+builder class. While it is possible to specify sort criteria using any class
+that implements the ``Bson`` interface, we recommend that you specify sort
+criteria through the ``Sorts`` builder. For more information on the benefits of
+using builders, see our :doc:`guide on builders </fundamentals/builders/>`. For
+more information on the ``Sorts`` builder class, see our
+:doc:`guide on the Sorts builder </fundamentals/builders/sort/>`.    
+
+For more information about the classes and interfaces in this section, see the
+following API documentation:
```

Comment:
This section actually does have a lot of "For more information,...". Maybe you could use a bulleted list or something like that to arrange this information.

Reply:
Thanks for the note. I do not understand what change you are suggesting, [as a bulleted list follows line 98](https://github.com/mongodb/docs-java/pull/66/files/a5dd2bb61d540bf9526f7792274a21d70c91f313#diff-b4955d68d249252f5472f742c01958cda1df25f3541a79865dd4d3e61314e836R101). Are you saying that a different part of this section could benefit from being formatted as a bulleted list?

Reply:
Note the above paragraph repeats "For more information..." several times. I'm wondering if there's a way to tidy it up a bit.

Reply:
Thanks for the clarification Chris.B and Chris.C. In order to both keep the link to the sorts guide closer to its context and mitigate the clunkiness caused by too much repetition of the phrase "For more information", I removed the link to the Builders guide. This positions the sorts builders link immediately after the `Sorts` class is mentioned, and prevents a triple "For more information". Let me know if you find this solutions satisfactory. Fix: https://github.com/mongodb/docs-java/pull/66/commits/57da0467c871d74a2d0596b17988856df84478dc